test/duct/middleware/web_test.clj

```diff
@@ -19,7 +19,14 @@
         (is (= (handler (mock/request :get "/")) response))
         (is (= @logs [[:info :duct.middleware.web/request
                        {:request-method :get, :uri "/"}]]))))
-    
+
+    (testing "synchronous with log level"
+      (let [logs    (atom [])
+            handler  (wrap-log-requests (constantly response) (->TestLogger logs) :trace)]
```

Comment:
Ensure that the `let` clauses are correctly aligned, and the lines are 80 characters or under.